README.md

```diff
@@ -13,9 +13,15 @@ jobs:
     name: Setup WordPress environment
     steps:
       - name: Start wp-env
-        uses: @godaddy-wordpress/setup-wp-env@v1
+        uses: godaddy-wordpress/setup-wp-env@v1
 ```
 
+### Usage with wp-env.json
+
+This Action assumes that no `wp-env.json` is present in the project that uses this Action in a workflow. If a `wp-env.json` file is present in the project, this Action config will override that config; however, if no inputs are provided that already exist in `wp-env.json`, those will be used instead (this is how `@wordpress/env` is designed).
```

Comment:
`.wp-env.json` (You are missing a dot at the beginning)

Reply:
Going to update this comment by adding the `.` and also changing the "error" to warning (because that's what @wordpress/env throws), although the environment does start.